.github/workflows/test.yml

```diff
@@ -0,0 +1,66 @@
+name: tests
+
+on:
+  pull_request:
+
+env:
+  python-version: "3.10"
+
+jobs:
+  security:
+    name: security
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
```

Comment:
if i remember correctly, `checkout` latest version is `v3` and `setup-python` should be `v4`